core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -1342,4 +1380,15 @@ private Path storeLog(String log, Path dirPath, String fileName) throws IOExcept
     public OpenShiftWaiters waiters() {
         return waiters;
     }
+
+    private static boolean isEqualOpenshiftConfig(OpenShiftConfig newConfig, OpenShiftConfig existingConfig) {
```

Comment:
Shouldn't be such `equal()` method comparing two OpenShiftConfig classes moved to OpenShiftConfig class?